marklogic-data-hub-central/ui/e2e/cypress/e2e/explore/graphEntitiesSettings.cy.tsx

```diff
@@ -224,9 +224,12 @@ describe("Entity Type Settings Modal", () => {
     dataModelDisplaySettingsModal.getModalBody().should("not.exist");
 
     cy.log("**Check in the sidebar that the entity type have the new color and icon**");
-    entitiesSidebar.getBaseEntity(defaultEntityTypeData.name).should("be.visible").and("have.attr", "data-color").then(color => {
-      expect(Cypress._.toLower(color)).equal(Cypress._.toLower(newEntityTypeData.color.HEX));
-    });
+    cy.waitUntil(() =>
```

Comment:
i thought @vmendez-marklogic deleted these. should we avoid waitUntil or not? 🤔

Reply:
The application had some waitUntil with a not useful implantation, we can use that function but in the correct way. But for the doubts @vmendez-marklogic can confirm

Reply:
For this scenario, the waitUntil will not make any difference, but this can be resolved with:

.and("have.attr", "data-color", {ExpectedValue}) => We can add to the assertion, not just the fact that the element has the attr, but also the expected value. That assertion will be asynchronous, so it is not needed the wait until

Example:

![image](https://github.com/marklogic/marklogic-data-hub/assets/126084579/a09099a2-1166-44c9-ac2a-f883a03d07ef)